src/steps/finalizeDependencies.ts

```diff
@@ -66,10 +66,10 @@ export async function finalizeDependencies(options: Options) {
 		await execaCommand(`npx all-contributors-cli generate`);
 		await removeDependencies(
 			["all-contributors-cli"],
-			(await readPackageData()).devDependencies,
+			(await readPackageData(".")).devDependencies,
 			"-D",
 		);
 	}
 
-	await execaCommand(`pnpm dedupe --offline`);
```

Comment:
I don't know why these errors are happening in CI 😞:

```plaintext
  \u2009ERR_PNPM_NO_OFFLINE_META\u2009 Failed to resolve @types/git-url-parse@9.0.3 in package mirror /home/runner/.cache/pnpm/metadata-v1.3/registry.npmjs.org/@types/git-url-parse.json
```

Reply:
Aha!

```plaintext
> husky

.git can't be found
```

It's that whatever is running scripts is probably not working in the right directory. Ha.